website/docs/docs/use-dbt-semantic-layer/sl-architecture.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "dbt Semantic Layer product architecture"
+id: sl-architecture
+description: "dbt Semantic Layer product architecture and related questions."
+sidebar_label: "Product architecture"
+---
+
+<VersionBlock firstVersion="1.6">
+
+At a high level, the dbt Semantic Layer has two primary components -- defining your metrics in MetricFlow and querying them with Semantic Layer APIs. These are made possible by the following architechtural pieces:
+
+| Components | Information | Developer plans | Team plans | Enterprise plans | License |
+| --- | --- | :---: | :---: | :---: | --- |
+| **[MetricFlow](/docs/build/about-metricflow)** | Use MetricFlow in dbt to centrally define your metrics. MetricFlow is a key component of the dbt Semantic Layer and is responsible for SQL query construction and defining specifications for dbt semantic models and metrics. | ✅ | ✅ |  ✅  | BSL package (code is source available) |
+| **MetricFlow Server**| A proprietary server that takes metric requests and generates and translates them to optimized SQL for the specific data platform| ❌ | ✅ | ✅ | Proprietary, Cloud (Team & Enterprise)|
```

Comment:
Open question.. should we add something about the metricflow CLI or can we just mention it as a part of metricFlow? @Jstein77

Reply:
I added a generic API section